api/camunda/camunda-openapi.yaml

```diff
@@ -0,0 +1,549 @@
+openapi: "3.0.3"
+info:
+  title: Camunda 8 REST API
+  version: "0.1"
+  description: API for communicating with the Camunda 8 cluster.
+  license:
+    name: Camunda License Version 1.0
+    url: https://github.com/camunda/camunda/blob/main/licenses/CAMUNDA-LICENSE-1.0.txt
+externalDocs:
+  description: Find out more
+  url: https://docs.camunda.io/docs/apis-tools/camunda-api-rest/overview/
+
+servers:
+  - url: "{schema}://{host}:{port}/v2"
```

Comment:
`/v2` but docs are `Version: 0.1`?

Reply:
Looks like we haven't paid attention to this little version attribute at all so far (also not in `/v1` 😅).
@pepopowitz, any suggestions here with all the versioning going on? 😄 
Is this `v2.0` currently? We'll then release that with Camunda `8.6`?
With `8.7` we'll then go to `v2.1`?
Or is this just `2.0` instead of `v2.0`?
So many options 😄

Reply:
Assuming there have been breaking changes in the API since its v1, I think `/v2` in the URL and `Version: Next`/`Version: 8.5` in the badge of the docs is the correct approach. It might seem strange, but from a user's perspective I think this results in the least chance for mistakes and code churn. 

Re: mistakes -- Our APIs rarely have breaking changes, but they do acquire additive changes with each product release. If an API has not had breaking changes, I wouldn't expect it to bump versions. On the other hand, if resources/properties are added in a new version of my Camunda cluster, I would like to read docs specific to my cluster version and see the exact API surface. 

Re: churn -- As a developer against Camunda, I would not want to need to change my code/environment variables to accommodate a new API URL depending on which version of a cluster I was using. I'd like to build against the `/v2` URL, and forget about it. I may find situations where I write code against the API and it doesn't work in the cluster I'm using, because it expects a resource or field that was introduced in a more recent version of the cluster, but I would prefer to deal with that very occasionally than to need to change the URL every time I'm hitting a different version of a cluster.

Having said that, I could see the case that we need to explain why there is a version in the API URL that seems to have no correlation to the version of my cluster. If either of you feel like that's important, please create an issue against the docs, because I think it's outside of the scope of this PR (and it would apply to every other API with an existing API Explorer.).

Reply:
Thanks, @pepopowitz, sounds good to me for the public docs to stick with `/v2` in the URL and `Version: Next`/`Version: 8.5` in the header. For the technical version of the OpenAPI file, I am still unsure what to put but that is outside of the scope of this PR here.